HackIllinois/UI/TableView/Cells/HIBubbleCell/HIEventCell.swift

```diff
@@ -37,21 +37,21 @@ class HIEventCell: HIBubbleCell {
         super.init(style: style, reuseIdentifier: reuseIdentifier)
 
         backgroundColor = UIColor.clear
-        favoritedButton.addTarget(self, action: #selector(didSelectFavoriteButton(_:)), for: .touchUpInside)
-        bubbleView.addSubview(favoritedButton)
-        favoritedButton.constrain(width: 58, height: 60)
-        favoritedButton.constrain(to: bubbleView, topInset: 0, trailingInset: 0)
-
         // add bubble view
         contentView.layer.backgroundColor = UIColor.clear.cgColor
         contentStackView.axis = .vertical
         contentStackView.translatesAutoresizingMaskIntoConstraints = false
         bubbleView.addSubview(contentStackView)
         contentStackView.leadingAnchor.constraint(equalTo: bubbleView.leadingAnchor, constant: 16).isActive = true
-        contentStackView.trailingAnchor.constraint(equalTo: favoritedButton.leadingAnchor).isActive = true
+        contentStackView.trailingAnchor.constraint(equalTo: bubbleView.trailingAnchor).isActive = true
```

Comment:
The trailing anchor of the content stack view should be relative to the favorite button's leading anchor. Otherwise, the favorite button may overlap with the contents of the stack view.